src/game/logic/object/weapon.cpp

```diff
@@ -457,3 +461,495 @@ bool Weapon::Is_Source_Object_With_Goal_Position_Within_Attack_Range(
+     { "LaserName", &INI::Parse_AsciiString, nullptr, offsetof(WeaponTemplate, m_laserName) },
+     { "LaserBoneName", &INI::Parse_AsciiString, nullptr, offsetof(WeaponTemplate, m_laserBoneName) },
+     { "PrimaryDamage", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_primaryDamage) },
+     { "PrimaryDamageRadius", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_primaryDamageRadius) },
+     { "SecondaryDamage", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_secondaryDamage) },
+     { "SecondaryDamageRadius", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_secondaryDamageRadius) },
+     { "ShockWaveAmount", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_shockWaveAmount) },
+     { "ShockWaveRadius", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_shockWaveRadius) },
+     { "ShockWaveTaperOff", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_shockWaveTaperOff) },
+     { "AttackRange", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_attackRange) },
+     { "MinimumAttackRange", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_minimumAttackRange) },
+     { "RequestAssistRange", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_requestAssistRange) },
+     { "AcceptableAimDelta", &INI::Parse_Angle_Real, nullptr, offsetof(WeaponTemplate, m_aimDelta) },
+     { "ScatterRadius", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_scatterRadius) },
+     { "ScatterTargetScalar", &INI::Parse_Real, nullptr, offsetof(WeaponTemplate, m_scatterTargetScalar) },
+     { "ScatterTarget", &WeaponTemplate::Parse_Scatter_Target, nullptr, offsetof(WeaponTemplate, m_scatterTarget) },
+     { "DamageType", &BitFlags<DAMAGE_NUM_TYPES>::Parse_Single_Bit_From_INI, nullptr, offsetof(WeaponTemplate, m_damageType) },
+     { "DeathType", &INI::Parse_Index_List, s_theDeathTypeNames, offsetof(WeaponTemplate, m_deathType) },
+     { "WeaponSpeed", &INI::Parse_Velocity_Real, nullptr, offsetof(WeaponTemplate, m_weaponSpeed) },
+     { "MinWeaponSpeed", &INI::Parse_Velocity_Real, nullptr, offsetof(WeaponTemplate, m_minWeaponSpeed) },
+     { "ScaleWeaponSpeed", &INI::Parse_Bool, nullptr, offsetof(WeaponTemplate, m_scaleWeaponSpeed) },
+     { "WeaponRecoil", &INI::Parse_Angle_Real, nullptr, offsetof(WeaponTemplate, m_weaponRecoil) },
+     { "MinTargetPitch", &INI::Parse_Angle_Real, nullptr, offsetof(WeaponTemplate, m_minTargetPitch) },
+     { "MaxTargetPitch", &INI::Parse_Angle_Real, nullptr, offsetof(WeaponTemplate, m_maxTargetPitch) },
+     { "RadiusDamageAngle", &INI::Parse_Angle_Real, nullptr, offsetof(WeaponTemplate, m_radiusDamageAngle) },
+     { "ProjectileObject", &INI::Parse_AsciiString, nullptr, offsetof(WeaponTemplate, m_projectileObject) },
+     { "FireOCL", &Parse_All_Vet_Levels_ASCII_String, nullptr, offsetof(WeaponTemplate, m_fireOCL) },
+     { "VeterancyFireOCL", &Parse_Per_Vet_Level_ASCII_String, nullptr, offsetof(WeaponTemplate, m_fireOCL) },
```

Comment:
`m_fireOCL` is correct here for "VeterancyFireOCL" ?

Same for several other Veterancy* names below.

Reply:
This is correct.